Makefile

```diff
@@ -37,7 +37,7 @@ build-native:
 	cargo build -p micro-rdk-server --bin micro-rdk-server-native
 
 native:
-	cargo run -p micro-rdk-server --bin micro-rdk-server-native
+	MICRO_RDK_WIFI_PASSWORD=0 cargo run -p micro-rdk-server --bin micro-rdk-server-native
```

Comment:
also for development convenience. used to bypass provisioning and use the local `viam.json`. 
will remove before merging